workflow/TPS.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env/bash
+
+#conda init bash
+echo "----------------------------starting script----------------------------"
+
+echo "-move to TPS base directory to run TPS"
+
+echo $0
+
+full_path=$(realpath $0)
+echo $full_path
+
+work_dir=$(dirname $full_path)
+echo $work_dir
+
+tps_dir=$(dirname $work_dir)
+echo $tps_dir
+
+cd $tps_dir
+
+# "defualt TPS params"
+./scripts/run \
```

Comment:
We still need to decide how we want to handle input arguments and whether this script wraps TPS + visualization or only visualization.

Reply:
This will be addressed by the config file #14